test/opt/test_pass_manager.cpp

```diff
@@ -94,6 +125,10 @@ TEST_F(PassManagerTest, Run) {
   AddPass<DuplicateInstPass>();
   AddPass<AppendOpNopPass>();
   RunAndCheck(text.c_str(), (text + "OpSource ESSL 310\nOpNop\n").c_str());
+
+  RenewPassManger();
+  AddPass<AppendMultipleOpNopPass>(3);
+  RunAndCheck(text.c_str(), (text + "OpNop\nOpNop\nOpNop\n").c_str());
```

Comment:
It would be nice to also see a use of `SinglePassRunAndCheck()` to make sure that `Args&&...` actually works. :)